india_compliance/patches.txt

```diff
@@ -1,5 +1,5 @@
 [pre_model_sync]
-india_compliance.patches.v15.check_version_compatibility
+india_compliance.patches.post_install.check_version_compatibility
```

Comment:
perhaps patch should go entirely & we should check this before all migration? it doesn't seem expensive to me.